extensions/google-java/src/test/kotlin/com/connectrpc/extensions/JavaErrorParserTest.kt

```diff
@@ -16,7 +16,7 @@ package com.connectrpc.extensions
 
 import com.connectrpc.AnyError
 import com.connectrpc.ConnectErrorDetail
-import com.connectrpc.google.rpc.Status
+import build.buf.google.rpc.Status
```

Comment:
Looks like this import just needs to be moved a few lines up to pass the lint check.

I also think we should update https://github.com/bufbuild/connect-kotlin/blob/5685a6a05faf183de32d2f597225f73a28bd733f/extensions/buf.gen.yaml#L4 to use `com.connectrpc` or `com.connectrpc.extensions` as the package prefix.